pages/support/[...path].vue

```diff
@@ -61,297 +61,238 @@
-          </p>
-        </Accordion>
-        <Accordion
-          title="Vous avez une question sur un organisme de formation ou une certification Qualiopi ?"
-        >
-          <p>
-            Notre équipe n'est pas en mesure de vous aider sur ces questions.
-            Si vous ne trouvez pas réponse à votre question dans les éléments suivants vous pouvez contacter <a
-              href="https://travail-emploi.gouv.fr/ministere/article/nous-contacter"
-              target="_blank"
-            >
-              le Ministère du Travail</a>.
-          </p>
-          <p>
-            <strong>Vous ne trouvez pas votre organismes de formation dans la liste ?</strong><br>
-            Retrouvez toutes les informations utiles pour déclarer votre organisme ou transmettre votre Bilan Pédagogique et Financier sur le site du Ministère du travail:
-            <a
-              href="http://travail-emploi.gouv.fr/formation-professionnelle/organismes-de-formation-fonctionnement/organismes-formation"
-              target="_blank"
-            >
-              les formalités de création et de fonctionnement des organismes de formation </a>.<br>
-          </p>
-          <p>
-            <strong>Vous souhaitez modifier certaines informations ?</strong><br>
-            Les informations sont issues des déclarations annuelles effectuées par l'organisme de formation auprès du services régional de contrôle de sa DREETS,
-            mais vous pouvez demander à tout moment la modification de certaines informations en contactant votre DREETS ou en accédant à votre espace personnel sur
-            <a
-              href="https://www.monactiviteformation.emploi.gouv.fr/"
-              target="_blank"
-            >l'application Mon Activité Formation</a>.<br>
-          </p>
-          <p class="mb-0">
-            <strong>Les informations concernant votre certification QUALIOPI vous semblent erronées ?</strong><br>
-            Contactez votre organisme certificateur afin qu'il transmette votre certification. Attention de bien lui fournir votre nouveau NDA si celui-ci a changé récemment.
-          </p>
-        </Accordion>
-        <Accordion
-          title="Vous rencontrez un problème avec DataPass ?"
-        >
-          <p class="mb-0">
-            DataPass est un service permettant aux administrations et collectivités d'être habilitées à accéder à des données en accès restreint. Que vous soyez une administration fournisseur de données ou utilisatrice de données, vous pouvez vous référez à la <a
-              href="https://datapass.api.gouv.fr/faq"
-              target="_blank"
-            >FAQ du service Datapass</a>.
-          </p>
-        </Accordion>
-      </AccordionGroup>
+<AccordionGroup @open="accordionOpened">
+
+  <!-- 🏢 Entreprise -->
+  <h3 class="text-2xl font-bold text-gray-title mt-10 mb-4">Entreprise</h3>
+  <Accordion title="Vos données sont visibles dans la base Sirene et vous souhaitez les rendre privées ?">
```

Comment:
I think Accordion title is `h3` as well, I guess it should be `h4`.
Done in https://github.com/datagouv/cdata/pull/391/commits/6b6e78a046c3fd6e7e9ca6057b0d2cefa3c0b319